doc/docs/master/how-tos/use-jupyterhub/using-jupyterhub.md

```diff
@@ -0,0 +1,236 @@
+# Using `python-pachyderm` in JupyterHub
+
+If you deployed JupyterHub as described in [Deploy Pachyderm with JupyterHub](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md),
+the `python-pachyderm` client is preinstalled in your JupyterHub instance
+This section describes a few basic operations that you can execute from JupyterHub
+to interact with Pachyderm.
+
+After you log in, use the [python-pachyderm](https://pachyderm.github.io/python-pachyderm/python_pachyderm.m.html#header-functions)
+client API to manage Pachyderm directly from your Jupyter notebook.
+
+You need to create a new Notebook and add your code in a new cell.
+To run your code, click the **Run** button.
+
+The following code initializes the Python Pachyderm client in JupyterHub:
+
+```python
+import python_pachyderm
+client = python_pachyderm.Client.new_in_cluster()
+```
+
+!!! note
+    This function is different from the function you'd call locally.
+
+For example, you can check the current user by
+running the following code:
+
+```python
+import python_pachyderm
+client = python_pachyderm.Client.new_in_cluster()
+print(client.who_am_i())
+```
+
+The following screenshot demonstrates how this looks in JupyterHub:
+![JupyterHub whoami](../../assets/images/s_jupyterhub_whoami.png)
+
+!!! note
+    If you have not enabled Pachyderm authentication, this
+    code returns an error.
+
+## Create a Pipeline
+
+As discussed in [Difference in Pipeline Creation Methods](#difference-in-pipeline-creation-methods),
+you can use the standard `create_pipeline` method or `create_python_pipeline`
+function to create Pachyderm pipelines in JupyterHub. Depending on your
+choice, use one of the following examples to create a pipeline:
+
+* By using the `create_python_pipeline` method:
+
+  ```python
+  import python_pachyderm
+  client = python_pachyderm.Client.new_in_cluster()
+
+  def relpath(path):
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), path)
```

Comment:
use consistent tabbing: either this should be 4 spaced tabbed, or the code below should be 2 space tabbed

Reply:
added 4 spaces.